### PR TITLE
Add support for IntelliJ IDEA 2016.1

### PR DESCRIPTION
**Pull request by jcgay**

### CODE REVIEW

File: mackup/applications/intellijidea-2016-1.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = IntelliJ IDEA 2016.1
+
+[configuration_files]
+Library/Application Support/IntelliJIdea2016.1
+Library/Preferences/IntelliJIdea2016.1
```
Comment (windwiny) [inline mackup/applications/intellijidea-2016-1.cfg]: The Community version name not include `IntelliJ`

    Library/Application Support/IdeaIC2016.1
    Library/Preferences/IdeaIC2016.1

Comment (jcgay) [inline mackup/applications/intellijidea-2016-1.cfg]: Ah thanks for the catch, I'm using the Ultimate edition.

**Comment by jcgay** [comment]
Do you prefer to have two separate configurations (ultimate and community edition) or just merge everything into a single one ?

**Comment by lra** [comment]
> Do you prefer to have two separate configurations (ultimate and community edition) or just merge everything into a single one ?

I think that would be great, care to do it?

**Comment by jcgay** [comment]
Sure, done!

**Pull request merged**